File: 3eme/worldfriendship symfony3.4/src/UserBundle/Entity/forum.php
<?php

namespace UserBundle\Entity;

use Cengizhan\ViewsCounterBundle\Model\VisitableInterface;
use Cengizhan\ViewsCounterBundle\Traits\VisitableEntityTrait;
use Doctrine\ORM\Mapping as ORM;

/**
 * forum
 *
 * @ORM\Table(name="forum")
 * @ORM\Entity(repositoryClass="UserBundle\Repository\forumRepository")
 */
class forum implements VisitableInterface
{
    use VisitableEntityTrait;


    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="title", type="string")
     */
    private $title;


    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string")
     */
    private $description;
    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="datetimetz")
     */
    private $date;
    /**
     *
     * @ORM\ManyToOne (targetEntity="categorie")
     * @ORM\JoinColumn (name="categorie",referencedColumnName="id", onDelete="CASCADE")
     *
     */
    private $categorie;
    /**
     *
     * @ORM\ManyToOne (targetEntity="User")
     * @ORM\JoinColumn (name="user",referencedColumnName="id")
     *
     */
    private $user;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @return string
     */
    public function getTitle()
    {
        return $this->title;
    }

    /**
     * @param string $title
     */
    public function setTitle($title)
    {
        $this->title = $title;
    }
    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }
    /**
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * @param \DateTime $date
     */
    public function setDate($date)
    {
        $this->date = $date;
    }
    /**
     * @return mixed
     */
    public function getCategorie()
    {
        return $this->categorie;
    }

    /**
     * @param mixed $categorie
     */
    public function setCategorie($categorie)
    {
        $this->categorie = $categorie;
    }
    /**
     * @return mixed
     */
    public function getUser()
    {
        return $this->user;
    }

    /**
     * @param mixed $user
     */
    public function setUser($user)
    {
        $this->user = $user;
    }
    public function __toString(){

        return (string) $this->id;

    }




}

File: 3eme/PiDevCodeNameOneTest/src/Entities/forum.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;



/**
 *
 * @author Moez
 */
public class forum {
    private int id;
    private int idCategorie;
    private int user;
    private String title;
    private String description;
    private String date;
    private Utilisateur Uti;

    public forum() {
    }

    public forum(int id, String title, String description,String date) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.date = date;
    }
    

    public forum(int id, int idCategorie, int user, String title, String description, String date) {
        this.id = id;
        this.idCategorie = idCategorie;
        this.user = user;
        this.title = title;
        this.description = description;
        this.date = date;
    }

    public forum(int idCategorie, int user, String title, String description, String date) {
        this.idCategorie = idCategorie;
        this.user = user;
        this.title = title;
        this.description = description;
        this.date = date;
    }

    public forum(int id, int idCategorie, Utilisateur Uti, String title, String description,String date) {
        this.id = id;
        this.idCategorie = idCategorie;
        this.Uti = Uti;
        this.title = title;
        this.description = description;
        this.date = date;
        
    }
    
    

    

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdCategorie() {
        return idCategorie;
    }

    public void setIdCategorie(int idCategorie) {
        this.idCategorie = idCategorie;
    }

    public int getUser() {
        return user;
    }

    public void setUser(int user) {
        this.user = user;
    }
    
    

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    @Override
    public String toString() {
        return "forum{" + "id=" + id + ", categorie=" + idCategorie + ", user=" + user + ", title=" + title + ", description=" + description + ", date=" + date + '}';
    }

   
    
    

    

    

    

    
    
}
